tests/llmcompressor/transformers/obcq/test_consecutive_runs.py

```diff
@@ -137,10 +138,15 @@ class TestConsecutiveRunsGPU(TestConsecutiveRuns):
     def setUp(self):
         from transformers import AutoModelForCausalLM
 
+        kwargs = {}
+        # if optimized self.model is passed, then must be using Linear Modules
+        if is_model_ct_quantized_from_path(self.model):
+            kwargs["quantization_config"] = self.quantization_config
```

Comment:
i know this is a test, but curious if we make use of `**kwargs` heavily in the code? i'm old and was taught this is not good practice, but huggingface does it everywhere. do we have any standardized guide for this sort of thing? An alternative would be something like

```python
        self.model = AutoModelForCausalLM.from_pretrained(
            self.model,
            device_map=self.device,
            quantization_config=self.quantization_config if s_model_ct_quantized_from_path(self.model) else None,
            )
```

Reply:
I never heard of kwargs being a bad actor before, but makes sense -- hard to keep in track of all the variables. 
We don't have a standard guide for this.
The alternate looks good, I just got rid of the kwargs so ill keep that in mind going forward!